rllib/algorithms/appo/tf/appo_tf_learner.py

```diff
@@ -139,16 +139,21 @@ def compute_loss_per_module(
             + (mean_kl_loss * self.curr_kl_coeffs_per_module[module_id])
         )
 
-        return {
-            self.TOTAL_LOSS_KEY: total_loss,
-            POLICY_LOSS_KEY: mean_pi_loss,
-            VF_LOSS_KEY: mean_vf_loss,
-            ENTROPY_KEY: -mean_entropy_loss,
-            LEARNER_RESULTS_KL_KEY: mean_kl_loss,
-            LEARNER_RESULTS_CURR_KL_COEFF_KEY: (
-                self.curr_kl_coeffs_per_module[module_id]
-            ),
-        }
+        # Register important loss stats.
```

Comment:
New pattern: compute_loss_per_module->single tensor
everything else goes into the new register metric(s) API.